ssl/record/ssl3_record.c

```diff
@@ -678,9 +678,7 @@ int tls1_enc(SSL *s, int send)
                      * we can't write into the input stream: Can this ever
                      * happen?? (steve)
                      */
-                    fprintf(stderr,
-                            "%s:%d: rec->data != rec->input\n",
-                            __FILE__, __LINE__);
+                    fprintf(stderr, "tls1_enc(): rec->data != rec->input\n");
```

Comment:
This is changing the code - personally I don't see the value in the use of __FILE__ and __LINE__ here either but it is a surprise to see a different behaviour given the description.